src/core/loader/load-collections.js

```diff
@@ -7,25 +7,24 @@ import { initCollection } from "../collections.js";
 import fp from "../../../lib/lodash/fp/fp.js";
 
 const getKey = item => item.collection;
-const loadToCache = (screen, path) => key =>
-    screen.load.json5({ key: `${path}items/${key}`, url: `${path}items/${key}.json5` });
+const loadToCache = screen => key => screen.load.json5({ key: `items/${key}`, url: `items/${key}.json5` });
 const getKeys = config => Object.values(config).map(getKey).filter(Boolean);
 
-export const loadCollections = (screen, config, path = "") => {
+export const loadCollections = (screen, config) => {
     const keys = getKeys(config).flat();
-    keys.forEach(loadToCache(screen, path));
+    keys.forEach(loadToCache(screen));
 
     return new Promise(resolve => {
         const cataloguesLoaded = () => {
-            keys.forEach(initCollection(screen, path));
+            keys.forEach(initCollection(screen));
             resolve();
         };
 
         const collectionsLoaded = () => {
             const catalogueKeys = fp
-                .uniq(keys.map(key => screen.cache.json.get(`${path}items/${key}`).catalogue))
+                .uniq(keys.map(key => screen.cache.json.get(`items/${key}`).catalogue))
```

Comment:
I guess this is fine actually, if we properly name debug collections so that they're unique to what the game actually uses

Reply:
Yeah that's the only issue - collections should be uniquely named anyway. Maybe we can look at this later and making it more robust.